src/js/Autocompletes/Autocomplete.js

```diff
@@ -457,14 +452,7 @@ export default class Autocomplete extends PureComponent {
     }
   }
 
-  componentWillUpdate(nextProps, nextState) {
-    if (this.state.isOpen !== nextState.isOpen) {
-      const menuFn = nextProps[`onMenu${nextState.isOpen ? 'Open' : 'Close'}`];
-      if (menuFn) {
-        menuFn();
-      }
-    }
-
+  componentWillReceiveProps(nextProps) {
```

Comment:
This should probably be in `componentWillUpdate` since it might run into the same problems as #300